src/test/java/com/ably/kafka/connect/integration/ChannelSinkTaskTest.java

```diff
@@ -40,6 +41,7 @@
  */
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@Disabled
```

Comment:
Why is this test disabled?  Do we still need it and it should be fixed, or is it no longer applicable?

Reply:
All the tests are related to the old logic of using realtime client. SinkTask should be covered in the integration test(separate PR).